Tropos/Formatters/TimeOfDay.swift

```diff
@@ -0,0 +1,52 @@
+import Foundation
+
+enum TimeOfDay {
+    case Morning
+    case Day
+    case Afternoon
+    case Night
+}
+
+extension NSCalendar {
+    func timeOfDay(forDate date: NSDate) -> TimeOfDay {
+        let components = self.components(.Hour, fromDate: date)
+
+        if components.hour < 4 {
+            return .Night
+        } else if components.hour < 9 {
+            return .Morning
+        } else if components.hour < 14 {
```

Comment:
It's possible this could be improved—would you be interested in raising an issue where we can discuss it? For now I believe this faithfully maintains the behaviour of the existing Objective-C version.